tools/integration-tests/tester/tests/testutil.go

```diff
@@ -205,21 +207,15 @@ func SendTransactionOnRandomPeer(t *testing.T, peers []*framework.Peer, addrBala
 
 // SendIotaTransaction sends sentValue amount of IOTA tokens and remainders from and to a given peer and returns the fail flag and the transaction ID.
 // Every peer sends and receives the transaction on the address of index 0 by default.
-// Optionally, the address index for from and to can be specified.
-func SendIotaTransaction(t *testing.T, from *framework.Peer, to *framework.Peer, addrBalance map[string]map[balance.Color]int64, sentValue int64, fromAndToAddressIndex ...uint64) (fail bool, txId string) {
+// Optionally, the nodes to pledge access and consensus mana can be specified.
+func SendIotaTransaction(t *testing.T, from *framework.Peer, to *framework.Peer, addrBalance map[string]map[balance.Color]int64, sentValue int64, fromAddressIndex uint64, toAddressIndex uint64, manaPledgeIDs ...identity.ID) (fail bool, txId string) {
```

Comment:
I would rather have `accessPledgeID` and `consensusPledgeID` as individual arguments, like you did for `from/toAddressIndex`. Alternatively, we could introduce a `TransactionConfig` or `TransactionParameters` struct that this function accepts as argument. When you instantiate such a config struct, fields you leave out will be the zero value for their type, which we can detect in the function and apply default values to the actual transaction. For example, leaving out `from/toAddressIndex` will init them to 0, which is actually the default here. Or when you leave out `pledgeIDs`, their `emptyID` field denotes to use the `ID` of the `from` node, and so on. What do you think?

Reply:
Good idea